docs/src/guide/best-practices.md

```
@@ -477,6 +477,63 @@ fgumi dedup --input sorted.bam --output deduped.bam \
+  --out1 out_R1.fastq.gz \
+  --out2 out_R2.fastq.gz \
+  --out0 /dev/null
+```
+
+Key points:
+- Set `--read-structures` in Step 1 to match where your UMIs actually live (e.g. `8M+T +T` for a UMI on R1 only). See [Read Structures](read-structures.md).
+- **DRAGEN-ready by default**: a duplex UMI stored as `RX:Z:AAAAAAAA-CCCCCCCC` becomes the read name `@readname:AAAAAAAA+CCCCCCCC` — identical to `samtools fastq -U`, and the `+`-joined format DRAGEN expects.
+- In split (`--out1`/`--out2`) mode the `/1` `/2` suffixes are omitted so R1 and R2 read names match, as in `samtools fastq -1/-2`. Reads that are neither cleanly R1 nor R2 go to `--out0` (or stdout if `--out0` is omitted).
+- Any `.gz`/`.bgz` output path is written as **BGZF** (gzip-compatible for downstream tools, yet block-indexable). Pass `--threads` to parallelize BGZF compression on compressed output.
+
+#### Optional: Interleaved output and custom delimiters
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)